src/codegen/templates/_library_interpreter.py.mako

```diff
@@ -63,3 +63,95 @@ class LibraryInterpreter(BaseInterpreter):
         raise NotImplementedError
 %endif
 %endfor
+
+    def read_power_i16(
```

Comment:
Why are these hand-written? Please add a comment explaining the reason. You can use a Mako comment `##` so it isn't included in the rendered output file.

Reply:
The reasons for the functions to be handwritten is that basically we don't have or have incomplete/incorrect data for these functions.
1) `ReadPowerBinaryI16/ReadPowerF64` - there is no data in `daqmxAPISharp.json` file
2) ReadRaw 
   - the datatype for readArray is generic in `daqmxAPISharp.json`
```
               {
               "dotNetDataType": "object[]", 
               "direction": "out", 
               "dotNetModifier": "", 
               "internalDescription": "", 
               "name": "readArray", 
               "cviBufferSizeIsExplicit": true, 
               "default": null, 
               "cviDataType": "object[]", 
               "internalType": "object[]"
            },
```
 - Also, expected c call is to pass `read_array.nbytes` but while autogenerating we get `read_array.size`
    
![image](https://user-images.githubusercontent.com/126869186/234595631-8bdfc935-4c80-4206-b6f2-17cab00ec6a8.png)

3) WriteRaw - the datatype of writeArray is incorrect in `daqmxAPISharp.json` 
```
            {
               "dotNetDataType": "object", 
               "direction": "in", 
               "dotNetModifier": "", 
               "internalDescription": "", 
               "name": "writeArray", 
               "cviBufferSizeIsExplicit": false, 
               "default": null, 
               "cviDataType": "object", 
               "internalType": "object"
            }, 
```

Reply:
Ok, please create a technical debt item for these.

Reply:
Created a [Technical Debt 2377787](https://dev.azure.com/ni/DevCentral/_workitems/edit/2377787): Autogenerate handwritten read/write methods in library interpreter to track this issue.